README.md

```diff
@@ -3,14 +3,16 @@
 # Project Name
 
 >My Portfolio project
+with Intro,works and about sections
 
 
 ## Built With
 
--Html
--CSS
--Using Flexbox
--Added Linters
+- Html
+- CSS
+- Using Flexbox
+- Using Grid
+- Added Linters
 
 ## Live Demo (if available)
```

Comment:
Hey @serialferris, you have done well implementing your readme file using the template provided😊, but it seems you have not properly customised it.

It would be really nice to add the following to the readme file:
- Project name
- Proper project description
- Information for users on how to clone and run the project locally

This will make your readme file look more professional 😁, you can check out this [template](https://github.com/microverseinc/readme-template/blob/master/README.md) to guide you.